### PR TITLE
Zigbee docs update

### PR DESCRIPTION
**Pull request by sanchosk**
Updated majority commands I could from the changelog with a lot of help from Discord and reading source code (where I could understand it).
Far from perfect, but more up to date comparing to current docs.

### CODE REVIEW

File: docs/Zigbee.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-!!! info "Zigbee2Tasmota serves as a gateway for devices connected to a Zigbee wireless network to bridge their communications over to Wi-Fi"
+<!-- Required extensions: pymdownx.betterem, pymdownx.tilde, pymdownx.emoji, pymdownx.tasklist, pymdownx.superfences -->
```
Comment (blakadder) [inline docs/Zigbee.md:1]: why is that here?

File: docs/Zigbee.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
...
+**Zigbee2Tasmota serves as a gateway for devices connected to a Zigbee wireless network to bridge their communications over to Wi-Fi**
```
Comment (blakadder) [inline docs/Zigbee.md:2]: restore formatting to what it was

File: docs/Zigbee.md [hunk 3]
```diff
@@ -452,18 +453,15 @@ Command `ZbLight` configures a Zigbee device to be Alexa controllable. Specify t
...
 ZbLight 0x1234,2
 ZbLight Kitchen_Light,1   (see ZbName)
 ```
 
 Once a light is declared, Z2T will monitor any change made to the light via Z2T or via remotes, either from a direct message or via a group message. Z2T will then send a read command to the light, between 200ms and 1000ms later, and memorize the last value.
 
-To read the last known status of a light, use `ZbLight <device>`
-
-!!! example
+To read the last known status of a light, use `ZbLight <device>`. Example:
```
Comment (blakadder) [inline docs/Zigbee.md:464]: restore formatting

File: docs/Zigbee.md (hunk 1 above)
Comment (sanchosk) [reply docs/Zigbee.md:1]: No idea, I think it might have been added by my editor tool :(

File: docs/Zigbee.md (hunk 2 above)
Comment (sanchosk) [reply docs/Zigbee.md:2]: Now I understand! My editor does not support the !!! info or !!! example... I have to redo it.

Comment (sanchosk) [reply docs/Zigbee.md]: Removed in latest commit. Restored the formatting, too.

File: docs/Zigbee.md (hunk 3 above)
Comment (sanchosk) [reply docs/Zigbee.md]: Restored original formatting.
Might be beneficial to align all the formattings, but I cannot find out any how-to for the !!! markdown usage...

File: docs/Zigbee.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-!!! info "Zigbee2Tasmota serves as a gateway for devices connected to a Zigbee wireless network to bridge their communications over to Wi-Fi"
+!!! info Zigbee2Tasmota serves as a gateway for devices connected to a Zigbee wireless network to bridge their communications over to Wi-Fi
```
Comment (barbudor) [inline docs/Zigbee.md:1]: You must leave the quotation mark otherwise the text is not shown anymore

Comment (sanchosk) [reply docs/Zigbee.md]: Provided new commit. Can you point me out to some documentation how to use the info tag? I tried to browse the md instructions, but none of them mentions such tag :(

Comment (barbudor) [reply docs/Zigbee.md]: Hi @sanchosk
There are a few links suggested from the [README.md](https://github.com/tasmota/docs/blob/development/README.md) in the Docs repo.
This one is specifically about [Admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions) as it is called.
But learned many things the hard-way, making changes and locally regenerating the pages to check the look and feel and validate the syntax. For anything more than just changing a few words/sentence, testing your change locally is a must-do.

Comment (sanchosk) [reply docs/Zigbee.md]: Thanks for the tip.
I was using local viewer called retext, but it did not understand the tags at all :(
If you look at the history of the commits, I removed them from the whole document and re-formatted it a bit, but those changes were rejected to keep it in line with the original layout.

**Comment by blakadder** [comment]
Why are examples referring to what I suppose is the friendly name of that device? What do all the MQT command outputs serve in each example?

**Comment by sanchosk** [comment]
> 
> 
> Why are examples referring to what I suppose is the friendly name of that device? What do all the MQT command outputs serve in each example?

In majority of the cases, the only difference after using setOption is the output format - e.g. payload of the output message gets changed.
Not sure how to show that without the output...

**Comment by sanchosk** [comment]
> 
> 
> Why are examples referring to what I suppose is the friendly name of that device? What do all the MQT command outputs serve in each example?

I wanted to provide real life example to the option outcome. I thought it would be more descriptive for the reader...

**Comment by barbudor** [comment]
@s-hadinger Do you think all those details are interresting ?

**Comment by blakadder** [comment]
examples not using default settings are probably the inverse of helpful to a novice.

**Comment by sanchosk** [comment]
Can you, plz, make one example so I know how to update the page correctly? It's a bit difficult to go up and down and I even forgot I did this...
I even moved to zigbee2mqtt during the time... But I still have the sonoff zigbee bridge, so I'll be able to get new examples if needed.

**Comment by sanchosk** [comment]
I understand you are busy, guys, but can you, plz, share some practical hints how to update the doc in a correct way? I'll be happy to do so - I believe the current docs are quite outdated and while my commit is far from perfect, I believe it would help a lot to people in the wild...

**Comment by ascillato** [comment]
Hi, the examples in this PR are very useful. What do you think about merging this?
